### ISSUE TITLE
Docs: export c-lightning-REST tls.cert (certificate.pem)

### ISSUE THREAD

**Issue by benjamin-wilson** [raspiblitz/raspiblitz#3371]
Currently LND has an 'Export' option under CONNECT > EXPORT to export macaroons & TLS.cert. When using CLN this option is not available.

**Comment by openoms** [comment]
CLN works differently. You might want to refer to the commando plugin documentation.

**Comment by benjamin-wilson** [comment]
When connecting Zeus to my node it displays a macaroon. It also requests a certificate for verification.

**Comment by openoms** [comment]
Yes, that is for C-LightningREST
get it again from CONNECT - MOBILE - CLREST

**Comment by benjamin-wilson** [comment]
Where can I get the TLS cert?

**Comment by openoms** [comment]
> Where can I get the TLS cert?

it is shown as a QR code when you go through. CONNECT as well but you don't really need it.

**Comment by benjamin-wilson** [comment]
I don't see a QR for it and it isn't mentioned anywhere either. (Going though ZEUS_CLREST option)

**Comment by benjamin-wilson** [comment]
Revisiting this issue, there is no option to export the TLS cert to prevent MITM attacks.

**Comment by openoms** [comment]
For mainnet the file is under:
```
/home/bitcoin/c-lightning-REST/bitcoin/certs
```
extract it as a string:
```
cert=$(grep -v 'CERTIFICATE' /home/bitcoin/c-lightning-REST/bitcoin/certs/certificate.pem | tr -d '=' | tr '/+' '_-' | tr -d '\n')
echo ${cert}
```
generate a QRcode with the shortcut in the admin user:
```
qr ${cert}
```

Related docs: https://github.com/Ride-The-Lightning/c-lightning-REST#security

Curious are you planning to use it with Zeus (import to Android?) or please describe the use case.

**Comment by benjamin-wilson** [comment]
Yes I am planning on importing it to Android for Zeus. When you scan the QR in Zeus it warns you to do this to prevent MITM. If I am not mistaken there were instructions in raspiblitz how to do this when I was using LND but not CLN. Thank you for the docs link.

**Comment by openoms** [comment]
It needs to be in DER format for Android to read the .crt file:
```
# convert to DER format
sudo -u bitcoin openssl x509 -outform der -in  /home/bitcoin/c-lightning-REST/bitcoin/certs/certificate.pem -out  /home/bitcoin/c-lightning-REST/bitcoin/certs/certificate.crt
```

Did not find a good way yet to transfer the file remotely. The QRcode does not work with binary and Tor is too broken for onionshare.

Trying:
https://github.com/claudiodangelis/qrcp
